### DISCUSSION TITLE
[Resolved] Vercel Open Graph tool missing paths and fail looking for DOM elements

### DISCUSSION THREAD

**Discussion by ThomasWT** [vercel/community#2282] [Bug Report]
Hello.

I have a Nuxt 3.4 deployment with SSR.

This tool is located https://vercel.com/{{username}}/{{project}}/{{deploymentid}}/og

Looking at the deployment, it only finds static pages and when asked to look at a specific path, does not find anything either. Even though the OG elements are present on SSR. I don't know if it's only for static site generation, but it does not say that anywhere or if it's only working for `@vercel/og`. Maybe it could look for a sitemap.xml and serve that as paths so it's useful for all sites.

![image](https://user-images.githubusercontent.com/1246641/235987923-82816b5a-3c56-4a32-b23b-04621bdd76fa.png)

![image](https://user-images.githubusercontent.com/1246641/235988048-db397464-2fb3-48d3-968b-d4006068430b.png)

**Comment by amyegan** [comment]
Thanks for reporting this, @ThomasWT! The team released a fix for the OG elements that weren't being recognized
